exp/operator/api/v1alpha4/conditions_consts.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha4
+
+import clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha4"
+
+const (
+	// PreflightCheckCondition documents a Provider that has not passed preflight checks.
+	PreflightCheckCondition clusterv1.ConditionType = "PreflightCheckPassed"
+
+	// MoreThanOneExistsReason (Severity=Info) documents that more than one instance of provider
+	// exists in the cluster
+	MoreThanOneExistsReason = "MoreThanOneExists"
+
+	// MoreThanOneExistsMessage a human readable message that explains PreflightCheckCondition.
+	MoreThanOneExistsMessage = "More than one instance of core provider exists in the cluster"
```

Comment:
This is currently specific to the core provider, do we need to include that in the name of the const so that we can have variations later for other types?

Reply:
This condition will be shared between providers, It's probably easier to remove "core" from the message.